tools/clean_duplicate_accounts.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/*.
+    require_module 'standard';
+.*/
+
+require_once(__DIR__."/../functions/functions.inc");
+
+$dependsField = 'dependentOnID';
+$do_update = false;
+
+
+function find_duplicates()
+{
+    $sql = <<<SQL
+        SELECT
+            *
+        FROM
+            `Members` AS base
+        WHERE
+            (
+            SELECT
+                COUNT(AccountID) AS COUNT
+            FROM
+                `Members`
+            WHERE
+                Email = base.Email
```

Comment:
MySQL is by default case insensitive for string based entries.

Reply:
...seriously?

MySQL is WEIRD. 😁  But it works in our favour here, so whatevs.